manifests/quick-start/base/minio/minio-deploy.yaml

```diff
@@ -30,7 +30,7 @@ spec:
           lifecycle:
             postStart:
               exec:
-                command: [mkdir, -p, /data/my-bucket]
+                command: ["mkdir", "-p", "/data/my-bucket", "&&", "mkdir", "-p", "/data/my-bucket-2", "&&", "mkdir", "-p", "/data/my-bucket-3"]
           readinessProbe:
             httpGet:
```

Comment:
wanted to be able to test with multiple buckets to make sure we're writing to and deleting from the right ones

Reply:
just moved this code to e2e manifests so it's no longer here